mmv1/products/compute/RouterNat.yaml

```diff
@@ -308,6 +308,17 @@ properties:
           - :ERRORS_ONLY
           - :TRANSLATIONS_ONLY
           - :ALL
+  - !ruby/object:Api::Type::Array
+    name: 'endpointTypes'
+    immutable: true
+    min_size: 1
+    description: |
+      Specifies the endpoint Types supported by the NAT Gateway.
+      Supported values include:
+            `ENDPOINT_TYPE_VM`, `ENDPOINT_TYPE_SWG`,
+            `ENDPOINT_TYPE_MANAGED_PROXY_LB`.
+    default_from_api: true
+    item_type: Api::Type::String
```

Comment:
Using a string type here instead of a enum (as the API specifies) to follow the same implementation used for the "sourceIpRangesToNat" field in this resource.